README.md

```diff
@@ -3,3 +3,8 @@
 ## Team Members
 - Griffin Brome | 63057285
 - Patrick Mahler | 29343597
+- Ren Lin | 35321405
+
+
+### Selection Sort
```

Comment:
Not sure what you meant to do here. But Bubble sort should be its own category. You have overwritten the description of Selection sort instead